api/document/js/__tests__/schema.test.ts

```diff
@@ -21,6 +21,14 @@ describe('unimplementedNode', () => {
   });
 });
 
+describe('inlineFootnote', () => {
+  it('is cool', () => {
```

Comment:
How about 'serializes to include a data-emblem attr'?

Reply:
LOL, my bad; I always start tests by naming them "is cool" and then circle back to give them a useful name later, but I forgot to do that this time!